specification.md

```diff
@@ -1446,6 +1446,10 @@ export interface TextDocumentClientCapabilities {
 				valueSet: CodeActionKind[];
 			};
 		};
+		/**
+		 * Whether code action supports flagging actions for `auto fix`
+		 */
+		isPreferred?: boolean;

```

Comment:
Can you please add a `Since 3.15.0` to the comment so that readers now since when this is in the spec.